services/distribution/src/main/resources/application.yaml

```diff
@@ -251,6 +251,8 @@ services:
       allow-list: ${DCC_VALIDATION_SERVICE_ALLOWLIST}
       allow-list-signature: ${DCC_VALIDATION_SERVICE_ALLOWLIST_SIGNATURE}
       allow-list-certificate: ${DCC_VALIDATION_SERVICE_ALLOWLIST_CERTIFICATE}
+      ccl-directory: ${COMMON_COVID_LOGIC:ccl}
+      ccl-allow-list: CCL-DE-0001
```

Comment:
this doesn't look like a list - is it really a list!?

Reply:
This is the way I saw that the list is set in application yaml. Ex: "supported-languages: DE, EN, BG, PL, RO, TR"